core/pom.xml

```diff
@@ -259,6 +259,41 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>prep-output-dir</id>
+            <goals>
+              <goal>exec</goal>
+            </goals>
+            <phase>compile</phase>
+            <configuration>
+              <executable>mkdir</executable>
+              <arguments>
+                <argument>-p</argument>
+                <argument>${project.build.directory}/generated-docs</argument>
+              </arguments>
+            </configuration>
+          </execution>
+          <execution>
+            <id>config-appendix</id>
+            <goals>
+              <goal>java</goal>
+            </goals>
+            <phase>compile</phase>
+            <configuration>
+              <mainClass>org.apache.accumulo.core.conf.ConfigurationDocGen</mainClass>
+              <classpathScope>compile</classpathScope>
+              <arguments>
+                <argument>--generate-markdown</argument>
+                <argument>${project.build.directory}/generated-docs/configuration-properties.md</argument>
```

Comment:
Yes, we still need to generate documentation for the configuration properties and copy it to the accumulo website repo.